keychain-sdk/example_keychain_test.go

```diff
@@ -18,21 +18,24 @@ func Main() {
 	}))
 
 	app := keychain.NewApp(keychain.Config{
-		Logger: logger, // not required, but recommended
+		BasicConfig: keychain.BasicConfig{
+			Logger: logger, // not required, but recommended
 
-		// setup the connection to the Warden Protocol node
-		ChainID:      "warden",
-		GRPCURL:      "localhost:9090",
-		GRPCInsecure: true,
+			// setup the connection to the Warden Protocol node
+			ChainID: "warden",
```

Comment:
let's use this chance to update to our new default chain id:
```suggestion
			ChainID: "warden_1337-1",
```